docs/readmes/orc8r/deploy_intro.md

```diff
@@ -52,22 +52,38 @@ involves the following
     - Security or functionality-critical
     - Updating to newer patch should be seamless, with no manual intervention required
 
-Major and minor releases are tagged off the master branch, then a patch branch is opened starting at that tag.
-Patch releases are tagged on the respective patch branch.
+Major and minor releases are tagged off the master branch, then a patch branch is opened starting at that tag. Patch releases are tagged on the respective patch branch.
 
 The current release schedule tags a new minor version 3-4 times per year. Patch releases are tagged on an on-demand basis.
 
 ## Deploying specific release
 
-To target a specific release, checkout the Magma repository's relevant release
-branch when building artifacts. This is also a great place to find relevant
-Terraform values.
+To target a specific release, checkout the Magma repository's relevant release branch when building artifacts. This is also a great place to find relevant Terraform values.
 
 Values for recent Orchestrator releases are summarized below
 
+### v1.6.0
+Verified with Terraform version `0.15.5`.
+
+- `v1.6` [patch branch](https://github.com/magma/magma/tree/v1.6)
+- `github.com/magma/magma//orc8r/cloud/deploy/terraform/orc8r-aws?ref=v1.6`
+Terraform module source
+- `1.5.23` Helm chart version
+- Additional notes
+    - `12.6` PostgreSQL target release, newer versions will likely work as well
```

Comment:
#7433 needs to be backported to the v1.6 release branch!!! Currently 1.6 is still deploying 9.6.15.

Reply:
I didn't realize that change hadn't been backported. I think we should leave it as-is for now (9.6), with a note in the deploy guide that you need to override the engine version and make it e.g. 12.6